bean/internal/adapter/controller/auth/auth.go

```diff
@@ -14,6 +14,34 @@ type Controller struct {
 	LoginView interfaces.LoginView
 }
 
+func (c *Controller) Authenticate(next http.Handler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		sessionToken, err := getSessionToken(r)
+		if err != nil || sessionToken == nil {
+			removeSessionTokenCookie(w)
+			http.Redirect(w, r, "/get-started", http.StatusSeeOther)
+			return
```

Comment:
Once we have logout, we should just redirect there and let it handle removing the session token cookie.

Reply:
e615fdc6f432390a68e4f07a9c6a8fecf1f3a4a3